src/gui/core/register_widget.hpp

```diff
@@ -53,4 +53,10 @@
  *
  * "Calls" REGISTER_WIDGET3(id_definition, id, nullptr)
  */
-#define REGISTER_WIDGET(id) REGISTER_WIDGET3(id##_definition, id, nullptr)
+#define REGISTER_WIDGET(id) REGISTER_WIDGET3(id##_definition, id, nullptr)                                             \
```

Comment:
Doesn't build with the declarations of the horizontal and grid listboxes since they have no respective classes.